ci/script.sh

```diff
@@ -531,15 +531,15 @@ main() {
 
         SiliconLabs)
             # #99 regression tests
-            test_svd SIM3C1x4_SVD
-            test_svd SIM3C1x6_SVD
-            test_svd SIM3C1x7_SVD
-            test_svd SIM3L1x4_SVD
-            test_svd SIM3L1x6_SVD
-            test_svd SIM3L1x7_SVD
-            test_svd SIM3U1x4_SVD
-            test_svd SIM3U1x6_SVD
-            test_svd SIM3U1x7_SVD
+            test_svd_for_target cortex-m https://raw.githubusercontent.com/posborne/cmsis-svd/master/data/SiliconLabs/SiM3_NRND/SIM3C1x4.svd
+            test_svd_for_target cortex-m https://raw.githubusercontent.com/posborne/cmsis-svd/master/data/SiliconLabs/SiM3_NRND/SIM3C1x6_SVD
+            test_svd_for_target cortex-m https://raw.githubusercontent.com/posborne/cmsis-svd/master/data/SiliconLabs/SiM3_NRND/SIM3C1x7_SVD
+            test_svd_for_target cortex-m https://raw.githubusercontent.com/posborne/cmsis-svd/master/data/SiliconLabs/SiM3_NRND/SIM3L1x4_SVD
+            test_svd_for_target cortex-m https://raw.githubusercontent.com/posborne/cmsis-svd/master/data/SiliconLabs/SiM3_NRND/SIM3L1x6_SVD
+            test_svd_for_target cortex-m https://raw.githubusercontent.com/posborne/cmsis-svd/master/data/SiliconLabs/SiM3_NRND/SIM3L1x7_SVD
+            test_svd_for_target cortex-m https://raw.githubusercontent.com/posborne/cmsis-svd/master/data/SiliconLabs/SiM3_NRND/SIM3U1x4_SVD
+            test_svd_for_target cortex-m https://raw.githubusercontent.com/posborne/cmsis-svd/master/data/SiliconLabs/SiM3_NRND/SIM3U1x6_SVD
+            test_svd_for_target cortex-m https://raw.githubusercontent.com/posborne/cmsis-svd/master/data/SiliconLabs/SiM3_NRND/SIM3U1x7_SVD
```

Comment:
wouldn't 

```suggestion
            test_svd SiM3_NRND/SIM3C1x4
            test_svd SiM3_NRND/SIM3C1x6
            test_svd SiM3_NRND/SIM3C1x7
            test_svd SiM3_NRND/SIM3L1x4
            test_svd SiM3_NRND/SIM3L1x6
            test_svd SiM3_NRND/SIM3L1x7
            test_svd SiM3_NRND/SIM3U1x4
            test_svd SiM3_NRND/SIM3U1x6
            test_svd SiM3_NRND/SIM3U1x7
```

be enough?

Reply:
No. It was my initial idea.